srv/salt/ceph/restart/grafana/default.sls

```diff
@@ -1,6 +1,8 @@
+{% if (salt.saltutil.runner('select.minions', cluster='ceph', roles='grafana') != []) %}
 restarting grafana:
   salt.state:
     - tgt: 'I@roles:grafana and I@cluster:ceph'
     - tgt_type: compound
     - sls: ceph.monitoring.grafana.restart
     - failhard: True
+{% endif %}
```

Comment:
I _think_ this would result in a 'empty rendering' error when there is no role-grafana.

Adding a test.nop is probably needed